tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "baseUrl": "./",
+    "paths": {
+      "*": ["src/*"]
```

Comment:
Anything in the `/src` folder can be referenced without the `/src` prefix.
e.g. `import ModalTemplate from 'components/ModalTemplate'`